KIF Tests/CollectionViewTests.m

```diff
@@ -59,6 +59,27 @@ - (void)testTappingLastItemAndSectionWithoutAnimation
 }
 
 
+- (void)testTappingLastAndFirstRowAtPositionWithAccessiblityIdentifier
+{
+    UICollectionViewCell *lastCell = [tester waitForCellAtIndexPath:[NSIndexPath indexPathForRow:-1 inSection:-1] inCollectionViewWithAccessibilityIdentifier:@"CollectionView Tests CollectionView" atPosition:UICollectionViewScrollPositionBottom];
+    CGPoint lastPosition = [lastCell.superview convertPoint:lastCell.frame.origin toView:nil];
+    XCTAssertEqual(lastCell.frame.origin.x, lastPosition.x);
```

Comment:
Is this a vertically scrolling collection view? If so, shouldn't you be checking here the `y` position (whether it aligns to the top/bottom of the collection view bounds)?

Reply:
OK I'm comparing the y now. I'm a bit fuzzy on UIView coordinate conversion, but this seems to work.

Reply:
LGTM